sympy/matrices/matrices.py

```diff
@@ -299,22 +299,106 @@ def _setitem(self, key, value):
             return
 
     def copy(self):
+        """
+        Returns the copy of a matrix.
+
+        Examples
+        ========
+
+        >>> from sympy import Matrix
+        >>> A = Matrix(2, 2, [1, 2, 3, 4])
+        >>> A.copy()
+        Matrix([
+        [1, 2],
+        [3, 4]])
+
+        """
         return self._new(self.rows, self.cols, self._mat)
 
     def trace(self):
+        """
+        Returns the trace of a matrix i.e. the sum of the
+        diagonal elements.
+
+        Examples
+        ========
+
+        >>> from sympy import Matrix
+        >>> A = Matrix(2, 2, [1, 2, 3, 4])
+        >>> A.trace()
+        5
+
+        """
         if not self.is_square:
             raise NonSquareMatrixError()
         return self._eval_trace()
 
     def inv(self, method=None, **kwargs):
+        """
+        Return the inverse of a matrix.
+
+        CASE 1: If the matrix is a dense matrix.
+
+        Return the matrix inverse using the method indicated (default
+        is Gauss elimination).
+
+        kwargs
+        ======
+
+        method : ('GE', 'LU', or 'ADJ')
+
+        Notes
+        =====
+
+        According to the ``method`` keyword, it calls the appropriate method:
+
+          GE .... inverse_GE(); default
+          LU .... inverse_LU()
+          ADJ ... inverse_ADJ()
+
+        See Also
+        ========
+
+        inverse_LU
+        inverse_GE
+        inverse_ADJ
+
+        Raises
+        ------
+        ValueError
+            If the determinant of the matrix is zero.
+
+        CASE 2: If the matrix is a sparse matrix.
+
+        Return the matrix inverse using Cholesky or LDL (default).
+
+        kwargs
+        ======
+
+        method : ('CH', 'LDL')
+
+        Notes
+        =====
+
+        According to the ``method`` keyword, it calls the appropriate method:
+
+          LDL .... inverse_LDL(); default
```

Comment:
@jksuom, is this okay ?

Notes for dense matrix

````
Notes
        =====

        According to the ``method`` keyword, it calls the appropriate method:

          GE .... inverse_GE(); default
          LU .... inverse_LU()
          ADJ ... inverse_ADJ()

````
Notes for sparse matrix

````
Notes
        =====

        According to the ``method`` keyword, it calls the appropriate method:

          LDL .... inverse_LDL(); default
          CH ... inverse_CH()
````

Earlier there were issues with the alignment. Now, for the last one in each case 3 dots are there.